sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVUtils.scala

```diff
@@ -76,7 +76,7 @@ object CSVUtils {
         headerNames.diff(headerNames.distinct).distinct
       }
 
-      row.zipWithIndex.map { case (value, index) =>
+      val header = row.zipWithIndex.map { case (value, index) =>
```

Comment:
I think we can modify the logic here to make sure we generate non-duplicate header in just one run?

Reply:
Thanks for your review, I modified the code.